### PR TITLE
update PR template

### LINKED ISSUE #1904: Update PR template to include scope approval on big issues

### LINKED ISSUE #1905: Update PR template to require manually tested approvals

### PR DESCRIPTION
**Pull request by fedekunze**
Closes #1905 
Closes #1904

_Description:_

❤️ Thank you!

---

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,17 +1,20 @@
 Closes #ISSUE
 
-_Description:_
+**Description:**
 
 <!-- Briefly describe what you're adding or fixing with this PR -->
 
-❤️ Thank you!
+Thank you! 🚀
```
Comment (faboweb) [inline PULL_REQUEST_TEMPLATE.md]: XD why?????

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
 
 ---
 
-<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺
-v                    Thanks for creating a PR!
-v    Before smashing the submit button please review the checkboxes
-☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->
+For contibutor:
```
Comment (faboweb) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
For contributor:
```

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,17 +1,20 @@
...
 
 - [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
 - [ ] Reviewed `Files changed` in the github PR explorer
+- [ ] Added snapshots of the view components
```
Comment (jbibla) [inline PULL_REQUEST_TEMPLATE.md]: doesn't our CI check this for us?

Comment (fedekunze) [reply PULL_REQUEST_TEMPLATE.md]: snapshots on the PR description, not of the HTML. Will update it

Comment (faboweb) [reply PULL_REQUEST_TEMPLATE.md]: screenshot not snapshot I suppose?

**Pull request merged**